### PR TITLE
Regression test with live branch

### PR DESCRIPTION
**Pull request by mingwli**
[AB#466544](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/466544)

\* kept test repo using master branch, for its live is way behind master

### CODE REVIEW

**Comment by live1206** [comment]
Remove this fallback since the branch is always specified
https://github.com/dotnet/docfx/blob/b40c5c52366c70fdeb236556b776a031e07889d1/test/docfx.RegressionTest/RegressionTest.cs#L203

File: azure-pipelines-v3.yml [hunk 1]
```diff
@@ -188,19 +188,19 @@ parameters:
   type: object
   default:
     azure-docs-pr:
-      params: https://github.com/MicrosoftDocs/azure-docs-pr --profile --timeout 100 --regression-rules
+      params: https://github.com/MicrosoftDocs/azure-docs-pr --profile --branch live --timeout 100 --regression-rules
```
Comment (yufeih) [inline azure-pipelines-v3.yml:191]: consider change the default to `live`

**Review by yufeih — APPROVED** [review]

File: azure-pipelines-v3.yml (hunk 1 above)
Comment (yufeih) [reply azure-pipelines-v3.yml:191]: And also remove all `--branch live` here

File: azure-pipelines-v3.yml [hunk 2]
```diff
@@ -210,38 +210,38 @@ parameters:
...
-      params: https://github.com/MicrosoftDocs/azure-docs-cli --timeout 50
+      params: https://github.com/MicrosoftDocs/azure-docs-cli --branch live --timeout 50
     dataprep-dotnet-pr:
-      params: https://github.com/MicrosoftDocs/dataprep-dotnet-pr --timeout 30
+      params: https://github.com/MicrosoftDocs/dataprep-dotnet-pr --branch live --timeout 30
     azure-mediaplayer-typescript:
-      params: https://github.com/MicrosoftDocs/azure-mediaplayer-typescript --timeout 30
+      params: https://github.com/MicrosoftDocs/azure-mediaplayer-typescript --branch live --timeout 30
     quantum-docs-private:
-      params: https://github.com/MicrosoftDocs/quantum-docs-private --timeout 30 --branch main
```
Comment (live1206) [inline azure-pipelines-v3.yml:235]: keep this

**Review by live1206 — APPROVED** [review]

**Pull request merged**